compiler/p/codegen/OMRInstOpCodeProperties.hpp

```diff
@@ -4770,7 +4770,7 @@
    /* .description =    "Store doubleword byte reverse indexed", */
    /* .prefix      = */ 0x00000000,
    /* .opcode      = */ 0x7C000528,
-   /* .format      = */ FORMAT_UNKNOWN,
+   /* .format      = */ FORMAT_RS_RA_RB_MEM,
```

Comment:
Well, it's a bit embarrassing that I missed this in the original move to the new binary encoder. :sweat_smile:

That being said, I generally have a policy of not defining formats for any new instructions here without adding corresponding tests to `fvtest/compilerunittest/p/BinaryEncoder.cpp` to ensure that the binary encoder unit tests remain complete.

Reply:
Fixed in 815ba08